js_modules/dagit/packages/core/src/resources/ResourceRoot.tsx

```diff
@@ -0,0 +1,208 @@
+    <Page>
+      <PageHeader
+        title={
+          <Box flex={{direction: 'row', alignItems: 'center', gap: 12}}>
+            <Heading>{resourceName}</Heading>
+          </Box>
+        }
+      />
+      <Loading queryResult={queryResult} allowStaleData={true}>
+        {({topLevelResourceOrError}) => {
+          if (topLevelResourceOrError.__typename !== 'TopLevelResource') {
+            return null;
+          }
+
+          const configuredValues = Object.fromEntries(
+            topLevelResourceOrError.configuredValues.map((cv) => [cv.key, cv.value]),
+          );
+
+          return (
+            <SplitPanelContainer
+              identifier="explorer"
+              firstInitialPercent={70}
+              firstMinSize={400}
+              height="calc(100% - 60px)"
+              first={
+                <Table>
+                  <thead>
+                    <tr>
+                      <th style={{width: 120}}>Key</th>
+                      <th style={{width: 90}}>Type</th>
+                      <th style={{width: 90}}>Value</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                    {topLevelResourceOrError.configFields.map((field) => {
+                      const defaultValue = field.defaultValueAsJson;
+                      const actualValue = configuredValues[field.name];
+
+                      const isDefault = defaultValue === actualValue;
+
+                      return (
+                        <tr key={field.name}>
+                          <td>
+                            <Box flex={{direction: 'column', gap: 4, alignItems: 'flex-start'}}>
+                              <strong>{field.name}</strong>
+                              <div style={{fontSize: 12, color: Colors.Gray700}}>
+                                {field.description}
+                              </div>
+                            </Box>
+                          </td>
+                          <td>{remapName(field.configTypeKey)}</td>
+                          <td>
+                            <Box flex={{direction: 'row', justifyContent: 'space-between'}}>
+                              <Tooltip content={<>Default: {defaultValue}</>} canShow={!isDefault}>
+                                {actualValue}
+                              </Tooltip>
+                              {isDefault && <Tag>Default</Tag>}
+                            </Box>
+                          </td>
+                        </tr>
+                      );
+                    })}
+                  </tbody>
+                </Table>
+              }
+              second={
+                <RightInfoPanel>
+                  <RightInfoPanelContent>
+                    <Box
+                      flex={{gap: 4, direction: 'column'}}
+                      margin={{left: 24, right: 12, vertical: 16}}
+                    >
+                      <Heading>{resourceName}</Heading>
+                    </Box>
```

Comment:
Seems like it might be redundant and unnecessary since you've already got the page header.

Reply:
I do think the repetition is kind of nice personally, similar to the job page (this is a rough take based on Josh's [loom](https://www.loom.com/share/e4a3ad1f3dc04cd7b2331c645728c78b))

<img width="735" alt="Screen Shot 2023-02-02 at 3 03 50 PM" src="https://user-images.githubusercontent.com/10215173/216470910-e32e1abb-8cdb-4a01-939d-3b97c0b10865.png">